mira/dkg/resources/ncit_slim.json

```diff
@@ -40865,12 +40936,12 @@
         }, {
           "pred" : "http://purl.obolibrary.org/obo/NCIT_P108",
           "val" : "Jino Chinese"
-        }, {
-          "pred" : "http://purl.obolibrary.org/obo/NCIT_P325",
-          "val" : "A Chinese person from the Jino ethnic group."
         }, {
           "pred" : "http://purl.obolibrary.org/obo/NCIT_P208",
           "val" : "CL937807"
+        }, {
+          "pred" : "http://purl.obolibrary.org/obo/NCIT_P325",
+          "val" : "A Chinese person from the Jino ethnic group."
```

Comment:
@cthoyt We might want to add sorting to avoid spurious diffs in ncit_slim.json when we update it.